PalasoUIWindowsForms/Progress/LogBox.cs

```diff
@@ -287,7 +287,7 @@ private void Write(Color color, FontStyle style, string msg, params object[] arg
 						rtfBoxForDelegate.SelectionColor = color;
 						rtfBoxForDelegate.SelectionFont = fnt;
 #endif
-						rtfBoxForDelegate.AppendText(string.Format(msg + Environment.NewLine, args));
+						rtfBoxForDelegate.AppendText(GenericProgress.SafeFormat(msg + Environment.NewLine, args));
```

Comment:
Is there an advantage to using this wrapper instead of the string extension FormatWithErrorStringInsteadOfException? I would go so far as to say that this method isn't enough about progress that GenericProgress should be offering it as a public method.